content/open_source_programs.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Open-Source Developer Programs
+---
+
+Improving battery technology is critical as we transition to a green economy. Battery modeling helps to achieve this by reducing costs and increasing reliability and safety. The PyBaMM package provides an open-source framework for physics-based battery models and simulations, with a growing focus on real-world experiments and battery degradation.
+While most of our development team works day-to-day on creating new models and better ways to solve them, the PyBaMM team also offers projects which do not require domain-specific knowledge under several open-source programs, which can be fantastic contributions to the general framework.
+
+To find out more about PyBaMM, you can visit our website [pybamm.org](/) or read our [paper](https://openresearchsoftware.metajnl.com/articles/10.5334/jors.309/).
+
+## Getting started
+
+We mostly communicate via Slack, so you should start off by [joining our Slack workspace](/slack/) and heading to the appropriate (`#gsoc-main`/`#outreachy-main`/`#gsod-main`) channel.
+
+A comprehensive set of [example notebooks](https://docs.pybamm.org/en/latest/source/examples/index.html) is available for becoming familiar with PyBaMM.
+Knowledge of battery physics or mathematical modeling is *not* required for any of the projects, but may be beneficial for some.
+
+All coding will be done in Python, so contributors should have some proficiency in Python, or another object-oriented programming language.
+Each project will follow test-driven development, and also involve writing clear documentation (via Sphinx) and examples (mainly in the form of jupyter notebooks) alongside the code. New contributors should make themselves familiar with this workflow.
+Finally, new contributors should read the [contributing guidelines](https://docs.pybamm.org/en/latest/source/user_guide/contributing.html) and be familiar with the basics of Git and GitHub.
+
+## Applications
+
+Students must submit proposals based on the ideas suggested below. The open-source program's governing organisation will decide how many projects are funded, it’s unlikely that every project will be funded. Selection for the projects will take into consideration, in strongly decreasing order of importance:
+
+- our interactions with you leading up to the project selection date
+- your project application
+- your background and experience
+
+Applicants must have at least 1 non-trivial contribution in PyBaMM for their application to be considered.
+
+We provide a [template for proposals](https://docs.google.com/document/d/1gER-yFt5_exHEu9Lx-jfrTH8I7QQXci8oDKH-5wgMys/edit?usp=sharing) for applicants to use, but it is not required to follow it exactly and modifications are encouraged as needed.
```

Comment:
I think this section as a whole would be better off when placed in the GSoC Home, considering it was written for that purpose. We could have a different section here instead, though I am not sure what exactly at this time.

Reply:
IMO the section can stay common for all the programs, but the template should definitely be moved.

Reply:
That makes sense as well. 

The sentence "Applicants must have at least 1 non-trivial contribution in PyBaMM for their application to be considered." may not be applicable for NumFOCUS Small Developer Grants though, since that is not a popular programme (side note: I did glance at what other projects are using these grants for, we should definitely apply!). We can move this sentence as well elsewhere